data-serving/README.md

```diff
@@ -1,18 +1,78 @@
 
 # All things data
 
-## Case schema updates
-### What needs to be changed & where
+[toc]
```

Comment:
I don't think this works out of the box for GitHub. :-(

Reply:
:'( that's so sad. it worked nicely in my VScode md preview so I just assumed. I'll just remove